test/unit/srv_option_handling.test.ts

```diff
@@ -0,0 +1,119 @@
+import { expect } from 'chai';
```

Comment:
The tests in the test/unit directory should follow the directory structure of the src folder (see the tests directory [README](https://github.com/mongodb/node-mongodb-native/blob/main/test/readme.md)): it looks like what this set of unit tests is trying to cover is the `resolveSRVRecord` method from the connection_string file, so we want to put this in the `connection_string.test.js` inside a `describe('resolveSRVRecord()' => { // })` block